src/Page/Submission/Team1/Algorithm.js

```diff
@@ -0,0 +1,185 @@
+// Sherman Grewal
+// Oliver Lingertat
+// Lucy Cheng
+// Navya Mehta
+import React, { Component } from 'react';
+import { render } from "react-dom";
+import StockStats from '../../../Data/stocks';
+
+class stockobj extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    keys: Object.keys(StockStats), 
+    dates: StockStats.map(function(x) {
+        return x.map(function(day) {
+          return day.Date;
+        });
+      }),
+    closeVal: StockStats.map(function(x) {
+        return x.map(function(day) {
+          return day.Close;
+        });
+      }),
+    sma19: this.sma19(),
+    ema19: this.ema19(),
+    ema39: this.ema39(),
```

Comment:
I have heard of people choosing 20-days or 50-days but why choose 19-days and 39-days?